cmd/zfs/zfs_main.c

```diff
@@ -727,6 +727,29 @@ finish_progress(char *done)
 	pt_header = NULL;
 }
 
+static int
```

Comment:
Can you add a block comment above this function saying something along the lines of "this function checks if the passed file descriptor refers to /dev/null".

Reply:
Maybe a more descriptive function name too?

Reply:
That wouldn't be a bad idea.  How about `is_dev_null()` since we're only checking against /dev/null.

Reply:
I actually patched it to be named `is_dev_files()` in my local copy since I also modified it to catch /dev/zero. If you'd prefer,  I'll change it back, but it seemed reasonable to catch both of the files that can cause this, even if writing to /dev/zero makes less sense.